debezium-core/src/main/java/io/debezium/transforms/ByLogicalTableRouter.java

```diff
@@ -272,9 +327,7 @@ private Struct updateEnvelope(Schema newEnvelopeSchema, Struct oldEnvelope) {
 
     private Struct updateValue(Schema newValueSchema, Struct oldValue) {
         final Struct newValue = new Struct(newValueSchema);
-        for (org.apache.kafka.connect.data.Field field : oldValue.schema().fields()) {
-            newValue.put(field.name(), oldValue.get(field));
-        }
+        oldValue.schema().fields().forEach(field -> newValue.put(field.name(), oldValue.get(field)));
```

Comment:
Surely a matter of taste, but don't think it's more readable than before. An actual downside though is that stepping through this loop in a debugger gets more difficult. Anyways, no need to revert, but I'd perfer to not change such things for the sake of it.